src/python/pyproject.toml

```diff
@@ -0,0 +1,94 @@
+# NOTE: pyproject.toml requires a up-to-date pip version (pip 23+)
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=68.0", "wheel>=0.37.1"]
+
+[project]
+name = "phenix-apps"
+dynamic = ["version"]  # see section below, [tool.setuptools.dynamic]
+description = "phenix user apps, schedulers, and Scorch components"
+readme = "README.md"
+requires-python = ">=3.8,<4.0"
+license = {file = "LICENSE"}
+authors = [
```

Comment:
Are we putting all contributors as authors? If not I'd suggest just rolling it all up into a single SNL author and adding a contributors list if necessary.

Reply:
I just copied what was in `setup.py`. @activeshadow , thoughts?